baseplate/observers/sentry.py

```diff
@@ -18,11 +23,19 @@
 from baseplate.lib import config
 from baseplate.observers.timeout import ServerTimeout
 
-
 if TYPE_CHECKING:
     from gevent.hub import Hub
 
 
+ALWAYS_IGNORE_EXCEPTIONS = (
+    ConnectionError,
+    HTTPError,
+    Timeout,
+    TException,
+    MemcacheError,
+)
```

Comment:
It's kinda weird, but I think we want to do this with stringy names of exceptions rather than the actual exception classes right now. Unfortunately, [raven<6.1 doesn't support doing it with classes](https://github.com/reddit/baseplate.py/commit/070cc2d654fb3fcc200e67c93a880282d0a6d096) and we can't bump a minimum version until we go to v2.0 baseplate (since it's a breaking change)

Reply:
Is there a way we can capture this in the tests?